api/v1alpha1/httproute_types.go

```diff
@@ -264,6 +276,14 @@ type ForwardToTarget struct {
 	//
 	// +optional
 	TargetPort *TargetPort `json:"targetPort" protobuf:"bytes,2,opt,name=targetPort"`
+
+	// Filter defines filters to apply to the request before forwarding it to
+	// a targetRef.
```

Comment:
The intention here is that these filters are executed in addition to the filters defined at the Route-level. Can we include that in this description to avoid confusion down the road?